src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,51 @@
 import React from 'react';
+import { Post } from '../../types/Post';
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+import './PostInfo.scss';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+type Props = {
+  post: Post;
+};
+
+export const PostInfo: React.FC<Props> = ({
+  post,
+}) => {
+  const {
+    title,
+    body,
+    user,
+    comments,
+  } = post;
+
+  return (
+    <>
+      {user
+      && (
+        <div className="PostInfo">
+          <div className="PostInfo__header">
+            <h3 className="PostInfo__title">
+              {title}
+            </h3>
+
+            <p>
```

Comment:
so move this check here

Reply:
done, removed useless `<></>` wrapper as well.